docs/user_manual/results.md

```diff
@@ -28,6 +32,7 @@ Heuristics will be shown in the UI and will be colour-coded based on the level o
 ![Heuristic](./images/heuristic2.png)
 
 ## Tags
+
```

Comment:
Can we change the arrow on line 42 to be like the new one in French? I think it looks nicer. 😁